adoc/deployment-azure.adoc

```diff
@@ -0,0 +1,204 @@
+== Deployment on Microsoft Azure
+
+Deployment on Microsoft Azure is currently a tech preview.
+
+.Preparation Required
+[NOTE]
+====
+You must have completed <<deployment.preparations>> to proceed.
+====
+
+You will use {tf} to deploy the whole infrastructure described in
+<<architecture-azure>>. Then you will use the `skuba` tool to bootstrap the
+{kube} cluster on top of it.
+
+
+[[architecture-azure]]
+=== AZURE Deployment
+
+The AZURE deployment created by our {tf} template files leads to the
+creation of the infrastructure described in the next paragraphs.
+
+==== Network
+
+All of the infrastructure is created inside of a user specified AZURE region.
+The resources are currently all located inside of the same availability
+zone. All the nodes are placed inside of the same virtual network, within the same
+subnet.
+
+Worker nodes are never exposed to the public internet. On the opposite
+each master nodes has a public IP address by default. This allows users to
+connect to them via ssh from their computers.
```

Comment:
This should be the case, but it has been reverted by the current PR. We have to revisit this statement once the skuba PR is merged.

Reply:
Reverted. worker node is no longer exposed.